packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json

```
@@ -2,11 +2,11 @@
   "billing_payment_method_add_title": "Ajouter un moyen de paiement",
   "billing_payment_method_add_back_btn_title": "Retourner à mes moyens de paiement",
   "billing_payment_method_add_choose_type": "Choix du moyen de paiement",
-  "billing_payment_method_add_selected_common_redirect": "En cliquant sur \"Ajouter\", vous allez être redirigé vers la plateforme de paiement sécurisé.",
+
+  "billing_payment_method_add_selected_common_redirect": "Vous allez être redirigé sur l'interface sécurisée de notre prestataire pour enregistrer une nouvelle carte bancaire.",
```

```suggestion
  "billing_payment_method_add_selected_common_redirect": "Vous allez être redirigé(e) sur l'interface sécurisée de notre prestataire pour enregistrer une nouvelle carte bancaire.",
```
I don't know what is Content best practice on this

---

FYI @marie-j , french "écriture inclusive" seems to be the rule in that case